lua/cfc_disconnect_interface/client/cl_ponger.lua

```diff
@@ -7,7 +7,7 @@ local PING_MISS = 2 -- How many pings can we miss on join?
 
 local API_TIMEOUT = 5 -- How often to call the api
 
-local lastPing
+local lastPong
```

Comment:
This is the intention, yeah?

Reply:
I've kind of taken pong to mean ping received, thus cl_ponger.lua. can change this if it's leading to confusion

Reply:
> I've kind of taken pong to mean ping received, thus cl_ponger.lua. can change this if it's leading to confusion

Disregard, yes it is :)